tests/components/webmin/test_init.py

```diff
@@ -0,0 +1,40 @@
+"""Tests for the Webmin integration."""
+from unittest.mock import patch
+
+from homeassistant.components.webmin.const import DOMAIN
+from homeassistant.components.webmin.coordinator import WebminUpdateCoordinator
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+
+from .conftest import TEST_USER_INPUT_REQUIRED
+
+from tests.common import MockConfigEntry, load_json_object_fixture
+
+# pytestmark = pytest.mark.usefixtures("mock_setup_entry")
+
+
+async def test_unload_entry(hass: HomeAssistant) -> None:
+    """Test successful unload of entry."""
+
+    entry = MockConfigEntry(
+        domain=DOMAIN, options=TEST_USER_INPUT_REQUIRED, title="name"
+    )
+    entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.webmin.config_flow.WebminInstance.update",
```

Comment:
```suggestion
        "homeassistant.components.webmin.coordinator.WebminInstance.update",
```

Shouldn't this be the coordinator?

Reply:
The question would be, how should I verify the user input? Currently I create a `WebminInstance` and try to connect to the device. Another option would be the create the coordinator and go through it.

Reply:
This is the setup of the integration itself, not the config flow. You want to patch the update function called by the coordinator, called in `_async_update_data`.

Reply:
This is still `"homeassistant.components.webmin.config_flow.WebminInstance.update"`, maybe forgotten to change? :)